Assets/Scripts/CoinManager.cs

```diff
@@ -67,8 +67,11 @@ private void AddCoin(int viewId)
 
         private IEnumerator RespawnTimer()
         {
-            yield return new WaitForSeconds(SecondsToRespawn);
-            mPhotonView.RPC("Respawn", PhotonTargets.All);
+            if (SecondsToRespawn >= 0)

```

Comment:
Ahh I see. For stuff like we should try to make note of it with a comment so other programmers know what's going on.